src/Commands/KeyVault/Secret/SecretCreateCommand.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using Azure.Security.KeyVault.Secrets;
+using AzureMcp.Arguments.KeyVault.Secret;
+using AzureMcp.Models.Argument;
+using AzureMcp.Models.Command;
+using AzureMcp.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Server;
+
+namespace AzureMcp.Commands.KeyVault.Secret;
+
+public sealed class SecretCreateCommand(ILogger<SecretCreateCommand> logger) : SubscriptionCommand<SecretCreateArguments>
+{
+    private readonly ILogger<SecretCreateCommand> _logger = logger;
+    private readonly Option<string> _vaultOption = ArgumentDefinitions.KeyVault.VaultName.ToOption();
+    private readonly Option<string> _secretNameOption = ArgumentDefinitions.KeyVault.SecretName.ToOption();
+    private readonly Option<string> _secretValueOption = ArgumentDefinitions.KeyVault.SecretValue.ToOption();
+
+    protected override string GetCommandName() => "create";
```

Comment:
Please rebase with main.  We update the BaseCommand interface. Make sure you add the title property as well and update the McpServerTool attribute with the title.